VictoriaAlbert/VictoriaAlbert/Photo.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  Photo.swift
+//  VictoriaAlbert
+//
+//  Created by Amber Spadafora on 11/10/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+class Photo {
+
+    var photoID: String
+    var object: String // example: "object": "Gimmel ring"
+    var date: String
+    var place: String
+    var title: String
+    
+    init(photoID: String, object: String, date: String, place: String, title: String) {
+        
+        self.photoID = photoID
+        self.object = object
+        self.date = date
+        self.place = place
+        self.title = title
+    }
+    
+    convenience init?(from dictionary: [String: Any]) {
+        if let object = dictionary["object"] as? String,
+            let date = dictionary["date_text"] as? String,
+            let place = dictionary["place"] as? String,
+            let title = dictionary["title"] as? String,
+            let photoID = dictionary["primary_image_id"] as? String {
+                self.init(photoID: photoID, object: object, date: date, place: place, title: title)
+        } else {
+            return nil
+        }
+        
+    }
+    
+    func createSmallImageUrl(photoID: String) -> String {
+        let firstSixChars: String = String(photoID.characters.prefix(6))
+        
+        
+        let imageBaseUrlString: String = "http://media.vam.ac.uk/media/thira/collection_images/"
+        
+        let imageIdUrlAddOn: String = "\(photoID + "_jpg_o.jpg")"
+        let smallImageUrlString: String = imageBaseUrlString + (firstSixChars + "/") + imageIdUrlAddOn
+        
+        return smallImageUrlString
+        
+       // http://media.vam.ac.uk/media/thira/collection_images/2006AM/2006AM3589_jpg_o.jpg
+        
+    }
+    
+  //  http://media.vam.ac.uk/media/thira/collection_images/2006AM/2006AM6763.jpg
+    
+    func createLargemageUrl(photoID: String) -> String {
```

Comment:
I like how you put these URL functions in the model. You could have combined them into one function. Also, you don't need to pass photoID because it's a member of the class. Lastly, since you did make two functions (one for large and small) and they don't need arguments, they could also have been computed properties.